build/npm/navigation-react-native/navigation.react.native.d.ts

```diff
@@ -316,6 +316,10 @@ export interface TabBarProps {
      * Indicates whether the tabs should be at the bottom
      */
     bottomTabs?: boolean;
+    /**
+     * Indicates whether the tab bar can be scrolled horizontally
+     */
+    scrollable?: boolean;
```

Comment:
Thanks for making the typings changes. Don't forget to copy them over to the [typings file in the test folder](https://github.com/grahammendick/navigation/blob/master/NavigationReactNative/test/node_modules/%40types/navigation-react-native.d.ts)

Reply:
Ok